Makefile

```diff
@@ -1,10 +1,13 @@
-.PHONY: proto
+DB_URL=postgres://postgres:12345@localhost:5432/postgres?sslmode=disable
+TEST_DB_URL=postgres://postgres:12345@localhost:5432/testpostgres?sslmode=disable
+
+.PHONY: migrate-up migrate-down create-migration proto
 
 proto:
 	protoc --proto_path=protobuf --go_out=proto --go_opt=paths=source_relative --go-grpc_out=proto --go-grpc_opt=paths=source_relative protobuf/*.proto
 
 run:
-	JWT_TOKEN_SECRET=my_secret_key REST_PORT=8080 GRPC_PORT=50052 ADDRESS=0.0.0.0 go run cmd/main.go
+	JWT_TOKEN_SECRET=my_secret_key REST_PORT=8080 GRPC_PORT=50052 ADDRESS=0.0.0.0 DB_HOST=localhost DB_PORT=5432 DB_USER=postgres DB_PASSWORD=12345 DB_NAME=postgres go run cmd/main.go
```

Comment:
I would advice reading env variables from actual environment. It's not a good habit to set protected vars (such as tokens and passwords) directly in your makefile or other files that stored in CVS. 
For example you could add all `.env` in your `.gitignore`, then have several of them to init required environment when you need one. (e.g. `local.env`, `dev.env`, `qa.env`, `integration_tests.env`, etc.)